src/native/Cargo.toml

```diff
@@ -1,25 +1,38 @@
 [package]
 name = "ddtrace-native"
+build = "build/main.rs"
```

Comment:
I don't see any `build/main.rs` anywhere, missed adding to the branch?

Reply:
Good catch, it was an early test which I never made it to work. Removing it.